web/static/ivre/controllers.js

```diff
@@ -380,13 +380,16 @@ ivreWebUi
 	    switch(status) {
 	    case "open": return "label-success";
 	    case "closed": return "label-danger";
-	    case "filtered": return "label-warning";
+	    case "filtered":
+	    case "open|filtered":
+		 return "label-warning";
 	    }
 	};
 	$scope.short_port_status = function(status) {
-	    if(status === "filtered")
-		return "fltred";
-	    return status;
+	    return {
+		"filtered": "fltred",
+		"open|filtered": "opn | flt"
```

Comment:
This remark is purely esthetic, but:
- filtered is mapped to fltred (why not plain "filtered" or just "flt"?)
- but `open|filtered` is mapped to `opn | flt` and not `opn | fltred`, or better `opn|fltred`.
Maybe you could switch to `flt` and `opn|flt`? I can't see the impactof this, though.